ui/index.d.ts

```diff
@@ -10,6 +10,7 @@ import { ServiceClientCredentials } from 'ms-rest';
 import { AzureEnvironment, AzureServiceClientOptions } from 'ms-rest-azure';
 import { Disposable, Event, ExtensionContext, FileChangeEvent, FileChangeType, FileStat, FileSystemProvider, FileType, InputBoxOptions, Memento, MessageItem, MessageOptions, OpenDialogOptions, OutputChannel, Progress, QuickPickItem, QuickPickOptions, TextDocument, TextDocumentShowOptions, ThemeIcon, TreeDataProvider, TreeItem, Uri } from 'vscode';
 import { AzureExtensionApi, AzureExtensionApiProvider } from './api';
+import { AzureLoginStatus } from './src/azure-account.api';
```

Comment:
I'm wondering if you should just reference the azure account directly. We try to avoid exposing anything related to "azure-account.api" in our external contract (aka this file) since those types are owned/managed by the Azure Account extension, not us.

Assuming the Azure Account extension is a direct dependency of your extension, it's actually quite simple. Our "loadAzureAccount" method is a bit more complex because we have some extensions where the Azure Account extension is an optional dependency. If it's a required dependency, you can safely assume the extension exists and is already activated when you get the exports.

Reply:
Oh I see what you mean. I did what you said and it works flawlessly. I'll close this then!